core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -337,30 +354,12 @@ func SendEther(db *gorm.DB, from, to common.Address, value assets.Eth, gasLimit
 	return etx, err
 }
 
-func newAttempt(ctx context.Context, ethClient eth.Client, ks KeyStore, config Config, etx models.EthTx, suggestedGasPrice *big.Int) (models.EthTxAttempt, error) {
+// TODO: Remove gas limit multiplier?
```

Comment:
the comment should be elsewhere? I can't see 'gas limit multiplier' in this function